src/Symfony/Component/Translation/Loader/CsvFileLoader.php

```diff
@@ -38,7 +34,7 @@ protected function loadResource(string $resource): array
         }
 
         $file->setFlags(\SplFileObject::READ_CSV | \SplFileObject::SKIP_EMPTY);
-        $file->setCsvControl($this->delimiter, $this->enclosure, $this->escape);
+        $file->setCsvControl($this->delimiter, $this->enclosure);
```

Comment:
```suggestion
        $file->setCsvControl($this->delimiter, $this->enclosure, '');
```

the empty string is not the default behavior in PHP 8.x, but that's the behavior we want as we want to disable the proprietary escaping mechanism. Thus, relying on the default value of PHP is deprecated in PHP 8.4, forcing to pass the value explicitly (precisely because they want to change the default behavior in PHP 9)